app/locales/en/translations.js

```diff
@@ -270,6 +271,10 @@ export default {
     cancel_booking_reason_title: "Reason for canceling order"
   },
 
+  my_dashboard: {
+    my_dashboard: "My Dashboard"
```

Comment:
Why do we need 2 translations for same text? i.e ** my_dashboard and myDashboard**

Reply:
yeah, bymistake done, will remove the other one.

Reply:
it'd be pretty cool if we could reference translations, e.g `my_dashboard: "@some.other.key"`  ^^